src/components/Atoms/LinkAtom/LinkAtom.tsx

```diff
@@ -1,15 +1,24 @@
+import type { Url } from "next/dist/shared/lib/router/router";
 import Link from "next/link";
-import {type  ReactNode } from "react";
+import type { MouseEventHandler, ReactNode } from "react";
+import "./custom.css";
 
 type Props = {
-
   // Add props here
-  href: string,
-  children: ReactNode,
-  className?: string
+  href: Url;
+  children?: ReactNode;
```

Comment:
follow this [convention docs](https://coal-blouse-907.notion.site/30-3-Component-with-children-Convention-d34b65ef326b48279adfcd8c6bf2cd98?pvs=4)

Do not use children Props, use PropsWithChildren
apply all Props alias that use children type